src/components/DegreeForm.svelte

```diff
@@ -0,0 +1,56 @@
+<script>
+    import { createEventDispatcher } from 'svelte'
+ 
+    const institutionList = ["University of Pittsburgh", "University of Pennslyvania", "Pennslyvanie State University"];

```

Comment:
let's fix the typos in Pennsylvania before merging : )

Reply:
Wait, I see, this is just a placeholder anyway, to be replaced by the actual list in the database. Yes? In which case, never mind!